lib/mind_the_gapp/mta/scheduled_arrival_time.ex

```diff
@@ -0,0 +1,44 @@
+defmodule MindTheGapp.Mta.ScheduledArrivalTime do
+  alias MindTheGapp.Repo
+  alias MindTheGapp.ScheduledArrivalTime
+  alias MindTheGapp.ScheduledArrivalTime, as: ScheduledArrivalTimeSchema
+
+  def is_data_loaded?() do
+    !(Repo.all(ScheduledArrivalTimeSchema) |> Enum.empty?)
```

Comment:
I could see this check becoming expensive to run once the table grows very big. Is there a faster way to check whether the data is loaded without retrieving the whole table first?

Reply:
Yes! Apparently, there's an `Repo.aggregate` function.

Reply:
Also this  https://hexdocs.pm/ecto/Ecto.Repo.html#c:exists?/2